sdk/resourcemanager/Azure.ResourceManager.Core/tests/Scenario/SubscriptionOperationsTests.cs

```diff
@@ -124,6 +124,22 @@ public async Task TestGetResourceGroupOpsLong(int length)
             }
         }
 
+        [RecordedTest]
```

Comment:
Add a test case for the Get method in SubscriptionOperations object

Reply:
Done